student/src/test/resources/performance-integration-student.properties

```diff
@@ -1,6 +1,6 @@
-datasource.jdbcUrl=jdbc:mysql://52.33.126.67/session
+datasource.jdbcUrl=jdbc:mysql://[URL]/session
 datasource.username=remoteuser
-datasource.password=change*me
+datasource.password=[PASSWORD]
```

Comment:
The current setup is to have the DB credentials in the code which is obviously not good.  So I omitted them for the PR but would like to follow what you all are doing in order to run the tests on TeamCity cleanly.  Figured I could get some details on that and follow that pattern if possible.